web/src/main/webapp/WEB-INF/pages/candidate_create.jsp

```diff
@@ -109,11 +109,9 @@
                                 <input type="text" class="input form-control" name="email" id="email" data-bind="valueWithInit: 'email'"
                                        value="${candidate.email}"
                                        placeholder="${i18nemail}"
-                                       data-error="<spring:message
-                                    code="candidate.error.email.empty"/>"
+                                       <spring:message code="candidate.error.email.empty" var="messValue"/>data-error="${messValue}"
```

Comment:
The `${messValue}` refers to the message with id `candidate.error.name.empty`, but the original message had the id `candidate.error.email.empty`. Please fix the incorrect variable reference.